src/loginServices.js

```diff
@@ -45,13 +46,16 @@ export function loadTranslations(store, locale) {
 }
 
 export function getLocale(okapiUrl, store, tenant) {
-  fetch(`${okapiUrl}/configurations/entries?query=(module=ORG and configName=locale)`,
+  fetch(`${okapiUrl}/configurations/entries?query=(module=ORG and configName=localeSettings)`,
```

Comment:
Why have you changed the key used in the configuration module?

Reply:
Oh, I see -- it's because you've changed the structure of the data that it carries. Well, that makes a certain kind of sense, but why are you shoving locale and timezone together into a single configuration record anyway? Why don't they each get their own configuration record?

Reply:
@MikeTaylor So, the timezonePicker is now added to the locale settings page along with the localePicker. And the `configManager` component that is present handles single value on submit. Hence, stored the locale and timezone under one config in a JSON structure. Something similar is being done here: https://github.com/folio-org/ui-circulation/blob/master/settings/CheckoutSettings.js

Reply:
It doesn't seem right to me that we're allowing our data-structures to be dictated by UI design. I would have preferred to use a simplest data structure, then write UI code that access it.

That said, I don't think it's a big deal; so if the relevant code is already written, let's run with that rather than doing the extra work to change it now.